.github/workflows/tools.yml

```
@@ -167,13 +167,22 @@ jobs:
               cat temp-output
               tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
               rm temp-output
+          - id: root-certificates
+            subsystem: crypto
+            label: crypto, notable-change
```

Do we usually do these as `semver-minor`? I'm not sure.

```suggestion
            label: crypto, notable-change, semver-minor
```

I'm also not sure if it's notable.

---

https://github.com/nodejs/node/pull/45490, https://github.com/nodejs/node/pull/40280 and https://github.com/nodejs/node/pull/35546 were not labelled https://github.com/nodejs/node/labels/semver-minor but were labelled https://github.com/nodejs/node/labels/notable-change. I kind of feel that listing the removed and/or added certificates should be in the release notes (hence https://github.com/nodejs/node/labels/notable-change).

---

> Do we usually do these as `semver-minor`? I'm not sure.

@nodejs/releasers @nodejs/crypto Thoughts? We haven't been labelling root certificates updates as semver-minor, but maybe we should be?

---

I would say no, but I understand if people think differently. To me this is more a bug fix (a certificate that would error before is then accepted). It doesn't add anything to the public API.

---

I'm in the semver-minor camp but with only one toe and it's not even my big toe. Call it +.1